animations/lib/src/basics/09_animated_switcher.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+
+double generateHeight() => Random().nextDouble() * 200;
+
+double generateWidth() => Random().nextDouble() * 200;
+
+Widget generateContainer() => Container(
+      key: GlobalKey(),
+      height: generateHeight(),
+      width: generateWidth(),
+      color: Colors.purple,
+    );
+
+class AnimatedSwitcherDemo extends StatefulWidget {
+  static String routeName = '/basics/09_animated_switcher';
+
+  _AnimatedSwitcherDemoState createState() => _AnimatedSwitcherDemoState();
+}
+
+class _AnimatedSwitcherDemoState extends State<AnimatedSwitcherDemo> {
+  Widget container;
+
+  void initState() {
+    container = generateContainer();
+    super.initState();
+  }
+
+  void changeWidget() {
+    setState(() {
+      container = generateContainer();
+    });
+  }
+
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(),
+      body: Center(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            //AnimatedSwitcher Widget is used to switch between different widgets with a
```

Comment:
nit: Add a space after `//` and check the 80 character line length.

You could also just cut this comment.